sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -290,7 +290,7 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
     // properties should override settings in extraOptions.
     this.extraOptions ++= properties.asScala
     // explicit url and dbtable should override all
-    this.extraOptions += (JDBCOptions.JDBC_URL -> url, JDBCOptions.JDBC_TABLE_NAME -> table)
+    this.extraOptions ++= Map(JDBCOptions.JDBC_URL -> url, JDBCOptions.JDBC_TABLE_NAME -> table)
```

Comment:
The API requires `TraversableOnce`, so here we can use `Seq` instead of `Map`?

Reply:
You're right~ Sure.